### DISCUSSION TITLE
could you please migrate quarkus operator to a new repository?

### DISCUSSION THREAD

**Discussion by absynth76** [keycloak/keycloak#14167] [Archive]
For clarity purpose, about issues, discussions and so on, maintainability would be better if keycloak operator would be located in a dedicated repository.

In that matter, why not have keep the current keycloak-operator repository, using another branch, until it's final ? 

example, there are more than 1000 issues logged on keycloak repository, it's really hard or even impossible to distinguish the one that are operator related only.

**Comment by vmuzikar** [comment]
@absynth76 Hello, thank you for the suggestion. The new Operator lives inside the main Keycloak repository due to its tight integration with the server. You can filter out any issues related to the Operator using [labels](https://github.com/keycloak/keycloak/issues?q=is%3Aopen+is%3Aissue+label%3Aarea%2Foperator).

The [repository with the legacy Operator](https://github.com/keycloak/keycloak-operator) will be archived after Keycloak 20 is released.

**Reply by msakho** [reply]
Where can we find instruction or Custom Resources for the quarkus Operator?
The documentation always leads me to the legacy Operator.
Thanks
